prep-html-syntax/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <body>
```

Comment:
Oops! You are missing your `head` element. Even though there is no content you need to put in it for this exercise, a `head` element is required for all HTML documents.

Reply:
I added a head element with no content, but when I tried to commit, there was an error message that a title is required.  So I titled it "prep-html-syntax". Hope that's okay. Thank you!